iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/HelmProcessorTest.java

```diff
@@ -102,103 +107,65 @@ void validateAndReadFilesShouldThrowExceptionIfValuesFileNotRead() throws IOExce
 
-    try (var ignored = Mockito.mockStatic(HelmFilesystemUtils.class)) {
-      var valuesFile = Mockito.mock(InputFile.class);
-      when(valuesFile.contents()).thenReturn("container:\n  port: 8080");
-      var files = Map.of("values.yaml", valuesFile);
-      when(HelmFilesystemUtils.additionalFilesOfHelmProjectDirectory(any())).thenReturn(files);
-      when(helmEvaluator.evaluateTemplate(anyString(), anyString(), any()))
-        .thenReturn(TemplateEvaluationResult.newBuilder().setTemplate("containerPort: 8080 #1").build());
-      var inputFile = Mockito.mock(InputFile.class);
-      when(inputFile.uri()).thenReturn(URI.create("file:///projects/chart/templates/foo.yaml"));
-      var inputFileContext = new InputFileContext(Mockito.mock(SensorContext.class), inputFile);
-
-      var result = helmProcessor.processHelmTemplate("foo.yaml", "containerPort: {{ .Values.container.port }}", inputFileContext);
-
-      assertEquals("containerPort: 8080 #1", result);
-    }
+    assertThat(mapOfFilesAndTheirContent).size().isEqualTo(2);
+    assertThat(mapOfFilesAndTheirContent.get("values.yaml")).isEmpty();
   }
 
   @Test
-  void shouldSkipHelmEvaluationIfHelmEvaluatorThrows() throws IOException {
-    when(helmEvaluator.evaluateTemplate(anyString(), anyString(), any())).thenThrow(new IllegalStateException("Failed to evaluate template"));
+  void evaluateHelmTemplateShouldNotThrow() throws IOException {
     var helmProcessor = getInitializedHelmProcessor();
+    var templateEvaluationResult = Mockito.mock(TemplateEvaluationResult.class);
+    var inputFile = Mockito.mock(InputFile.class);
+    String path = "path";
+    String content = "content";
+    Map<String, String> templateDependencies = new HashMap<>();
+
+    when(inputFile.uri()).thenReturn(URI.create("file://" + "chart/templates/foo.yaml"));
+    when(helmEvaluator.evaluateTemplate(path, content, templateDependencies)).thenReturn(templateEvaluationResult);
 
-    try (var ignored = Mockito.mockStatic(HelmFilesystemUtils.class)) {
-      var valuesFile = Mockito.mock(InputFile.class);
-      when(valuesFile.contents()).thenReturn("container:\n  port: 8080");
-      var files = Map.of("values.yaml", valuesFile);
-      when(HelmFilesystemUtils.additionalFilesOfHelmProjectDirectory(any())).thenReturn(files);
-      var inputFileContext = mockInputFileContext("chart/templates/foo.yaml");
-
-      assertThatThrownBy(() -> helmProcessor.processHelmTemplate("foo.yaml", "containerPort: {{ .Values.container.port }}", inputFileContext))
-        .isInstanceOf(ParseException.class)
-        .hasMessage("Failed to evaluate Helm file chart/templates/foo.yaml: Template evaluation failed");
-    }
+    assertDoesNotThrow(() -> {
+      helmProcessor.evaluateHelmTemplate(path, inputFile, content, templateDependencies);
+      verify(helmEvaluator).evaluateTemplate(path, content, templateDependencies);
+    });
```

Comment:
This test is not good as you complitly mock the output. It like `var a = true; assertTrue(a);` Please think about what should be the goal of the test case.